### ISSUE TITLE
Image proxying does not proxy inline image previews

### ISSUE THREAD

**Issue by laCour** [mattermost/mattermost-server#8222]
#### Summary
With image proxying enabled, when a user sends a message with an insecure image link (i.e. not markdown), the image preview will load without proxying.

- **mattermost-server:** `release-4.7` @ `654fc2f`
- **mattermost-webapp:** `release-4.7` @ `685cab4`

#### Steps to reproduce

1. Enable image proxying.
2. Send a message with an insecure image link: `http://i.redd.it/6cohydckk6501.jpg`

#### Expected behavior
The image preview should be loaded using the configured image proxy.

#### Observed behavior (that appears unintentional) 
The original source image will be loaded for a preview image.

```
Mixed Content: The page at '--' was loaded over HTTPS, but requested an insecure image 'http://i.redd.it/6cohydckk6501.jpg'. This content should also be served over HTTPS.
```

**Comment by lindy65** [comment]
Thanks for the report @laCour,

I've created [this ticket](https://mattermost.atlassian.net/browse/PLT-8717) to investigate.

**Comment by ccbrown** [comment]
This'll be fixed in 4.7.

**Comment by lindy65** [comment]
I'll close this issue off for now and we can reopen if it reproduces after v4.7 releases on Feb 16th.
